src/app/wallet/wallet.component.ts

```diff
@@ -14,6 +14,10 @@ export class WalletComponent implements OnInit {
   hasUnminedCreatorCoins: boolean;
   showTransferredCoins: boolean = false;
 
+  sortedFromHighToLow: number = 0;
+  sortedPriceFromHighToLow: number = -1;
```

Comment:
I think 1 should signify sort ascending, -1 should signify sort descending, and 0 means not sorting on this column.  It's a bit more intuitive that way.